### PR TITLE
Bug 1503826 - Change redeploy check, using deploy name instead of runtime-name

### PR DESCRIPTION
**Pull request by rubenvp8510**
@josejulio Could you please review?

//cc @burmanm

### CODE REVIEW

**Comment by josejulio** [comment]
Sorry, forgot to fill in this before.

With this fix and after following the steps on the BZ I get a different runtime-name than the one specified (the filename).

I'm not sure if this is related to this fix (possible) or the Bundle script used to test.
con-el-fix

**Comment by rubenvp8510** [comment]
@josejulio  I fixed that issue, this was happening because the redeploy operation didn't set any runtime-name, so by default the runtime-name is the filename.

File: modules/plugins/jboss-as-7/src/main/java/org/rhq/modules/plugins/jbossas7/Redeployer.java [hunk 1]
```diff
@@ -59,7 +61,10 @@ boolean deploymentExists() {
 
     Result redeployOnServer() {
         Operation op = new Operation("full-replace-deployment", new Address());
-        op.addAdditionalProperty("name", runtimeName);
+        op.addAdditionalProperty("name", name);
+        if (runtimeName != null) {
+            op.addAdditionalProperty("runtime-name", runtimeName);
```
Comment (rubenvp8510) [inline modules/plugins/jboss-as-7/src/main/java/org/rhq/modules/plugins/jbossas7/Redeployer.java]: @josejulio I added the runtime-parameter here, if it is specified (not-null) I'll pass as a parameter of the full-replace-deployment op.

**Comment by josejulio** [comment]
LGTM

**Pull request merged**